src/hotspot/share/utilities/ostream.hpp

```diff
@@ -100,7 +100,7 @@ class outputStream : public ResourceObj {
    void print_raw(const char* str, size_t len)   { write(str,         len); }
    void print_raw_cr(const char* str)         { write(str, strlen(str)); cr(); }
    void print_raw_cr(const char* str, size_t len){ write(str,         len); cr(); }
-   void print_data(void* data, size_t len, bool with_ascii);
+   void print_data(void* data, size_t len, bool with_ascii, bool rel_addr=true);
```

Comment:
This allows for hex dumps to show absolute addresses which is important for comparing a buffer [written in Java](https://github.com/openjdk/jdk/pull/9305/files#diff-d290f5a7a572ba7d4f0c7d234eb73de6537a2aa1d02726527d8db0df23c874b7R118-R134) and [read in C++](https://github.com/openjdk/jdk/pull/9305/files#diff-5e4a9ef468fc0f1564d6cfbe98f44b1099842098366cbec4103710137b2022edR34-R36).